File: C#/SchoolSystem/SchoolSystem.Core/QueryFilters/CourseFilter.cs
namespace SchoolSystem.Core.QueryFilters
{
    public class CourseFilter
    {
        public int? StudentId { get; set; }
        public int? TeacherId { get; set; }
        public string Name { get; set; }
        public int? CourseId { get; set; }
        public string CourseName { get; set; }

    }
}

File: Java/schoolsystem/src/main/java/org/com/schoolsystem/core/queryFilters/CourseFilter.java
package org.com.schoolsystem.core.queryFilters;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class CourseFilter {

    private int studentId;
    private int teacherId;
    private String name;
    private int courseId;
    private String courseName;
}
